src/config/dynamic-theme-fixes.config

```diff
@@ -2483,6 +2483,11 @@ dash.cloudflare.com
 INVERT
 a[data-testid^="link-homepage"]
 
+CSS
+.monaco-editor-background, .monaco-editor .margin {
+    background-color: rgb(24 27 29) !important;
```

Comment:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

Reply:
@Gusted --darkreader-neutral-background does not match the theme of the website sadly (background becomes way too dark in comparison)

Reply:
It's recommended to still use this, because when users changes their colors due to brightness/contrast/sepia etc. the variable will change according to it, whereby this is static and will be out of place.

Reply:
Ah, is there a way to mix/manipulate the --darkreader-neutral-background color, similar to scss color modules?

Reply:
Not really, it's controlled by the user settings, however you could switch to `${originial color}` eg `${white} ` will turned to be black-ish.

Reply:
Ah, that gave me an idea. I flipped the rgb color, and set that as the original, and it looks good now!